# Publish all joint states -----------------------------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

# Position Controllers ---------------------------------------
joint1_position_controller:
  type: effort_controllers/JointPositionController
  joint: base_link__link_01
  pid: {p: 2000.0, i: 100, d: 500.0}
joint2_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_01__link_02
  pid: {p: 50000.0, i: 100, d: 4000.0}
joint3_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_02__link_03
  pid: {p: 20000.0, i: 100, d: 4000.0}
joint4_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_03__link_04
  pid: {p: 20000.0, i: 100, d: 4000.0}
joint5_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_04__link_05
  pid: {p: 20000.0, i: 100, d: 4000.0}
joint6_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_05__link_06a
  pid: {p: 20000.0, i: 100, d: 4000.0}
joint6a_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_06a__link_06
  pid: {p: 20000.0, i: 100, d: 2000.0}
joint7_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_06__link_07
  pid: {p: 20000.0, i: 100, d: 2000.0}
joint8_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_06__link_08
  pid: {p: 20000.0, i: 100, d: 2000.0}



joint9_position_controller:
  type: effort_controllers/JointPositionController
  joint: base_link__link_09
  pid: {p: 20000.0, i: 100, d: 4000.0}
joint10_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_09__link_10
  pid: {p: 50000.0, i: 100, d: 4000.0}
joint11_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_10__link_11
  pid: {p: 20000.0, i: 100, d: 4000.0}
joint12_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_11__link_12
  pid: {p: 20000.0, i: 100, d: 4000.0}
joint13_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_12__link_13
  pid: {p: 20000.0, i: 100, d: 4000.0}
joint14_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_13__link_14a
  pid: {p: 20000.0, i: 100, d: 4000.0}
joint14a_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_14a__link_14
  pid: {p: 20000.0, i: 100, d: 2000.0}
joint15_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_14__link_15
  pid: {p: 20000.0, i: 100, d: 2000.0}
joint16_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_14__link_16
  pid: {p: 20000.0, i: 100, d: 2000.0}
